### DISCUSSION TITLE
Request compatibility for GL-MT3000

### DISCUSSION THREAD

**Discussion by iceman3k** [xchwarze/wifi-pineapple-cloner#4] [General]
Requesting compatibility for GL-MT3000. Thank you.

**Comment by xchwarze** [comment]
Unfortunately, this development can only be ported to platforms that meet these requirements:
1. Architecture mips or mipsel 
2. Ported to OpenWrt 19

The above device does not comply with any of the following

**Reply by iceman3k** [reply]
Thank you for reply an info.
